src/System.Private.ServiceModel/tests/Benchmarks/PerfUsingCoreWCFService/CoreWCFPerfService/CoreWCFPerfService.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
```

Comment:
Use the Sdk `""Microsoft.NET.Sdk.Web"` and remove the package reference to Microsoft.AspNetCore

Reply:
If change to "Microsoft.NET.Sdk.Web",  visualstudio will throw IndexOutOfRangeException exception